app/views/home/index.html.erb

```diff
@@ -37,7 +41,63 @@
       <a href="#" class="button tiny">Search</a>
     </li>
 </li>
+</ul>
+</div>
 
 
-</ul>
+<div class="home-pics">
```

Comment:
I'd like to eventually iterate over all of our locations in the database and create these divs